utils/wasm-gen/src/config/syscalls.rs

```diff
@@ -137,18 +137,19 @@ impl SyscallsConfig {
         &self.precise_syscalls_config
     }
 
-    /// Error processing config for fallible syscalls.
+    /// Get error processing config for fallible syscalls.
     pub fn error_processing_config(&self) -> &ErrorProcessingConfig {
         &self.error_processing_config
     }
 }
 
 /// Syscall destination choice.
 ///
-/// `gr_send*` and `gr_exit` syscalls generated from this crate can be sent
-/// to different destination in accordance to the config.
-/// It's either to the message source, to some existing known address,
-/// or to some random, most probably non-existing, address.
+/// `gr_send*`, `gr_exit` and other message sending syscalls generated
+/// from this crate can send messages to different destination
+/// in accordance to the config. It's either to the message source,
+/// to some existing known address, or to some random, most probably
+/// non-existing, address.
 #[derive(Debug, Clone, Default)]
 pub enum SyscallDestination {
```

Comment:
Doc-comments for variants

Reply:
done in [8689748](https://github.com/gear-tech/gear/pull/3627/commits/86897483589f2b104ee8aa1a97b1fd8994a5dd72)